.github/workflows/synnax.build.yaml

```diff
@@ -110,24 +35,13 @@ jobs:
       - name: Create Release
         if: steps.filter.outputs.changed == 'true' && github.ref == 'refs/heads/main'
         id: create-release
-        uses: actions/github-script@v6
+        uses: softprops/action-gh-release@v1
```

Comment:
I dont recall making this change but I don't know whether it makes a difference?